README.md

```diff
@@ -158,9 +166,599 @@ search.start();
 - If you use Vue, check out [meilisearch-vue](https://github.com/meilisearch/meilisearch-vue/)
 - If you use Angular, check out [meilisearch-angular](https://github.com/meilisearch/meilisearch-angular/)
 
-## Compatibility with MeiliSearch
+## API resources
+
+List of all the components that are available in [instantSearch](https://github.com/algolia/instantsearch.js) and their compatibilty with [MeiliSearch](https://github.com/meilisearch/meilisearch-react/).
+
+### ✅ InstantSearch
+
+[instantSearch references](https://www.algolia.com/doc/api-reference/widgets/instantsearch/js/)
+
+`instantSearch` is the main component. It manages the widget and lets you add new ones.
+
+- ✅ IndexName: Name of your index. _required_
+- ✅ SearchClient: Search client, in our case instantMeiliSearch. See [customization](#customization) for details on options. _required_
+- ❌ numberLocale: Did not work with both algoliasearch and instantMeiliSearch.
```

Comment:
```suggestion
- ❌ numberLocale: Does not work with both Algoliasearch and InstantMeiliSearch.
```

Reply:
"Did" is confusing, the users don't have the context of your investigation when reading the README.